### DISCUSSION TITLE
Dependency versioning issue in gradle

### DISCUSSION THREAD

**Discussion by nagarajui7** [renovatebot/renovate#10891] [Request Help]
Hi Team,

We are looking for gradle based dependencies using renovate. The document suggested it will follow gradle official versioning policies. However, we are facing an issue with a particular dependency.

We have a dependency with versions `1.1.3-7` and `1.3-branch-team-1`.  

We are using `1.1.3-7` in our code, but renovate says `1.3-branch-team-1` is the latest version. 

As document says,  `A version with an extra non-numeric part is considered lower than a version without: 1.1.a < 1.1`  but here renovate it is not happening. 

Can you help me on this

**Comment by rarkins** [comment]
Did you typo one of the versions?

**Reply by nagarajui7** [reply]
@rarkins , no the first one(`1.1.3-7`) is created from master branch and the other one(`1.3-branch-team-1`) is created from a dev branch

**Reply by rarkins** [reply]
`1.3-branch-team-1` sorts higher than `1.1.3-7` in Renovate's gradle versioning, by design, and as far as I'm aware according to the Gradle specification too (your example of `1.1.a` and `1.1` is not the same as here). You can use `allowedVersions` in a package rule if you need to change this.
